### ISSUE TITLE
Steps don't have enough contrast ratio

### ISSUE THREAD

**Issue by kingamagyar** [productkind/monorepo#79]
## Expected behaviour
Expected contrast ratio of 4.5:1

## Actual Behaviour
Element has insufficient color contrast of 2.58 (foreground color: #02bb1d, background color: #ffffff, font size: 10.5pt (14px), font weight: normal). 

<img width="1048" height="643" alt="Image" src="https://github.com/user-attachments/assets/a3c6c012-654f-4008-afff-f2f99579817f" />
